src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRowMapper.java

```diff
@@ -38,6 +38,7 @@ public String mapRow(ResultSet rs, int rowNum) throws SQLException {
 		discreteGroundWater.timezoneCode = rs.getString("timezone_code");
 		discreteGroundWater.verticalDatumCode = rs.getString("vertical_datum_code");
 		discreteGroundWater.parameterCode = rs.getString("parameter_code");
+		discreteGroundWater.displayResult = rs.getString("display_result");
 
```

Comment:
there used to be a condition here that assigned the display_result to one of the values that gets written in the row writer. Is that to be omitted now in favor of a single value column?

Reply:
That condition was based on the parameter codes you pulled from the USGS page, though.  And then Jim preferred that we pull the parameter codes from the transform db, which resulted in the duplication of effort and possible confusion.

My thinking here is that the rdb provides the parameter code and the raw value (display_result) and Janell's code interprets it accordingly.   I just put this PR up to facilitate discussion in tomorrow's meeting ...

Reply:
good. then I recommend the below condition be revised.

Reply:
The RDB that currently comes from the legacy system has two columns with result data, and I am are trying to maintain that structure.

Reply:
yes, that is correct. But it turns out that there will need to be a change because of the multiple datums sent. We can simplify both sides. I will highlight the details on the discussion today.